appveyor.yml

```diff
@@ -57,6 +57,7 @@ install:
   - "set HOME=%APPVEYOR_BUILD_FOLDER%"
   - "%PIP% install setuptools pip --upgrade"
   - "%PIP% install check-manifest coveralls nose pytest pytest-cov twine wheel"
```

Comment:
Any reason for not using the previous line?

Reply:
None what so ever.